src/Lykke.Quintessence.Core.Abstractions/Blockchain/IWalletGenerator.cs

```diff
@@ -0,0 +1,9 @@
+using System.Threading.Tasks;
+
+namespace Lykke.Quintessence.Core.Blockchain
+{
+    public interface IWalletGenerator
+    {
+        Task<(string Address, string PrivateKey)> GenerateWalletAsync();
```

Comment:
Maybe, it's better to implement full contract and add `AddressContext` property to result.